framework/scripts/wazuh_server.py

```diff
@@ -436,7 +436,7 @@ async def monitor_server_daemons(loop: asyncio.BaseEventLoop, server_process: ps
     """
     comms_api_config = CentralizedConfig.get_comms_api_config()
     process_children = {
-        MANAGEMENT_API_DAEMON_NAME: 3,
+        MANAGEMENT_API_DAEMON_NAME[:15]: 3,
```

Comment:
This was needed because `psutil` is limitting the process name to 15 characters

```console
[psutil.Process(pid=1459, name='wazuh-server.py', status='sleeping', started='17:36:43'), psutil.Process(pid=1460, name='wazuh-server.py', status='sleeping', started='17:36:43'), psutil.Process(pid=1463, name='wazuh-engine', status='disk-sleep', started='17:36:43'), psutil.Process(pid=1537, name='wazuh-comms-api', status='sleeping', started='17:36:45'), psutil.Process(pid=1577, name='wazuh-server-ma', status='sleeping', started='17:36:47')]
```